util/storage-samples/storage-rv-sample/src/test/java/org/fidoalliance/fdo/storage/RvsStorageTest.java

```diff
@@ -186,13 +186,13 @@ void importAllowDenyListTest() {
 
       server = Server.createTcpServer(args).start();
 
+      final String configFilePath = "src/main/resources/config.properties";
```

Comment:
Move this file to src/test/resources/ folder.

Place a copy at component-samples/demo/rv/